### PR TITLE
Bug-fix on accordion arrow

### PR DESCRIPTION
**Pull request by nadijagraca**
## Description
Bug fix on accordion arrow:
- replaced loading image url for chevron down arrow with unicode for chevron down
- adjusted the padding and font color of accordion-header to match figma design
- added hover selector on accordion-header (font color highlighted only)

## Screenshot
Screenshot 2023-09-20 at 16 22 14

## Checklist

- [x] I have not referenced individuals, products or companies in any commits, directly or indirectly
- [x] I have not added data or restricted code in any commits, directly or indirectly
- [ ] I have updated the docstring of any public function/class/model changed
- [ ] I have added the PR number to the change description in the changelog fragment, e.g. `Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))` (if applicable)

## Types of changes

- [ ] Docs/refactoring (non-breaking change which improves codebase)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: vizro-core/changelog.d/20230920_155943_nadija_ratkusic_graca_accordion_arrow_demo_fix.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+- A bullet item for the Deprecated category.
+
+-->
+<!--
+### Fixed
+
+- A bullet item for the Fixed category.
+
+-->
```
Comment (maxschulz-COL) [inline vizro-core/changelog.d/20230920_155943_nadija_ratkusic_graca_accordion_arrow_demo_fix.md:36]: Can we add a fix here?

**Review by maxschulz-COL — APPROVED** [review]
Lgtm! Could you add a description of the bug-fix though?

File: vizro-core/src/vizro/static/css/accordion.css [hunk 2]
```diff
@@ -42,35 +42,37 @@
...
+  padding-right: 6px;
 }
 
 .accordion-button:not(.collapsed)::after {
   background-position: left;
-  transform: rotate(180deg);
-  float: left !important;
-  filter: var(--fill-accordion-button);
-  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 24 24' fill='none'><path fill-rule='evenodd' clip-rule='evenodd' d='M8.49845 9L7 10.5016L11.9984 15.5016L17 10.5016L15.4984 9L11.9984 12.5016L8.49845 9Z' fill='white' fill-opacity='0.6'/></svg>");
+  transform: scaleX(1.3) rotate(180deg);
```
Comment (petar-qb) [inline vizro-core/src/vizro/static/css/accordion.css:58]: I like this transformation 🥇.

**Review by petar-qb — APPROVED** [review]
Looks good!

File: vizro-core/src/vizro/static/css/accordion.css [hunk 3]
```diff
@@ -42,35 +42,37 @@
   overflow-anchor: none;
   background-color: inherit;
   letter-spacing: -0.014px;
-  padding-left: 0;
+  padding-top: var(--spacing-04);
```
Comment (huong-li-nguyen) [inline vizro-core/src/vizro/static/css/accordion.css:45]: If you add the padding here, can you remove the gap in the accordion class? Otherwise the space above and below the accordion title is not the same (see your screenshot on SUMMARY). Or is this intended?

**Review by huong-li-nguyen — CHANGES_REQUESTED** [review]
Thanks for fixing! 🚀 Did we tests this already on the deployed version if that fixes the bug? Could you double-check the top padding? See comment below

File: vizro-core/src/vizro/static/css/accordion.css (hunk 3 above)
Comment (huong-li-nguyen) [reply vizro-core/src/vizro/static/css/accordion.css:45]: Apparently, it's intended given the designs - if we could just incorporate the hover color as in the designs, then I think we should be good

File: vizro-core/src/vizro/static/css/accordion.css [hunk 4]
```diff
@@ -42,35 +42,38 @@
...
   padding-bottom: var(--spacing-04);
```
Comment (huong-li-nguyen) [inline vizro-core/src/vizro/static/css/accordion.css:46]: Could you double-check the height of the accordion buttons: https://www.figma.com/file/Raf81HNv2pUdvJ8IvE74yL/QB-DS_(v1.0.0)?node-id=17904%3A112724&mode=dev

They seem to have a height of 40px in the figma designs?

**Review by huong-li-nguyen — DISMISSED** [review]
Given that we are currently close to the Figma design, I've fine with the changes 👍 I'll let @Joseph-Perkins decide whether we actually want to follow the designs or not.

**Review by petar-qb — APPROVED** [review]

File: vizro-core/src/vizro/models/_page.py [hunk 5]
```diff
@@ -162,10 +162,10 @@ def _create_theme_switch():
     def _create_control_panel(controls_content):
         keyline = html.Div(className="keyline")
         control_panel = html.Div(
-            children=[*controls_content, keyline],
+            children=[keyline, *controls_content, keyline],
             className="control_panel",
         )
-        return control_panel if controls_content else None
+        return control_panel if controls_content else keyline
```
Comment (huong-li-nguyen) [inline vizro-core/src/vizro/models/_page.py:168]: If we only have one page and no controls there will be random keyline on the left-side no? Did we try to add it at the end of the nav panel container instead of the controls container?

File: vizro-core/src/vizro/static/css/accordion.css [hunk 6]
```diff
@@ -130,6 +131,10 @@ div.page_container .accordion_button:focus {
...
 
 div.page_container .accordion_button:hover {
-  background-color: var(--background-hover);
+  background-color: var(--state-overlays-selected-hover);
+  color: var(--text-primary);
+}
+
+div.page_container .accordion-button:hover {
   color: var(--text-primary);
 }
```
Comment (huong-li-nguyen) [inline vizro-core/src/vizro/static/css/accordion.css:140]: Remove this? You already added it above

File: vizro-core/src/vizro/static/css/token_names.css [hunk 7]
```diff
@@ -14,11 +18,11 @@
   --status-light-mode-error: rgba(240, 59, 58, 1);
   --status-light-mode-information: rgba(0, 158, 255, 1);
   --surfaces-dark-mode-bg-01: rgba(35, 38, 50, 1);
-  --surfaces-dark-mode-bg-02: rgba(27, 30, 42, 1);
+  --surfaces-dark-mode-bg-02: #1b1e2a;
   --surfaces-dark-mode-bg-03: rgba(20, 23, 33, 1);
   --surfaces-dark-mode-bg-card: rgba(35, 38, 50, 1);
   --surfaces-light-mode-bg-01: rgba(255, 255, 255, 1);
-  --surfaces-light-mode-bg-02: rgba(245, 246, 246, 1);
+  --surfaces-light-mode-bg-02: #f5f6f6;
```
Comment (huong-li-nguyen) [inline vizro-core/src/vizro/static/css/token_names.css:25]: Can we actually revert the changes here? The rgbas are the existing hex colors and we're using rgba throughout the entire file, I would just stick with it to be consistent

**Review by huong-li-nguyen — CHANGES_REQUESTED** [review]

File: vizro-core/src/vizro/models/_page.py (hunk 5 above)
Comment (nadijagraca) [reply vizro-core/src/vizro/models/_page.py:168]: Added at the end of the nav_panel. I had to remove nav_panel gap in order for the last accordion-header(title) to look symmetrical to the other accordion headers.

File: vizro-core/src/vizro/static/css/accordion.css (hunk 6 above)
Comment (nadijagraca) [reply vizro-core/src/vizro/static/css/accordion.css:140]: It is actually different class, accordion-button and accordion_button. Accordion-button refers to accordion titles, and accordion_button refers to page buttons. 
So accordion-button is 'ANALYSIS' button and accordion_button is 'Homepage'  button.

**Review by huong-li-nguyen — APPROVED** [review]
Amazing - thank you so much!!! 🚀

**Review by huong-li-nguyen — APPROVED** [review]

File: vizro-core/changelog.d/20230920_155943_nadija_ratkusic_graca_accordion_arrow_demo_fix.md [hunk 8]
```diff
@@ -31,7 +31,7 @@ Uncomment the section that is right (remove the HTML comment wrapper).
 
 ### Fixed
 
-- Fixed issue of accordion arrow not loading on deployed demo version
+- Fixed issue of accordion arrow not loading on deployed demo version ([#44](https://github.com/mckinsey/vizro/pull/44))
```
Comment (maxschulz-COL) [inline vizro-core/changelog.d/20230920_155943_nadija_ratkusic_graca_accordion_arrow_demo_fix.md:34]: 😍

**Review by petar-qb — APPROVED** [review]
It's completely ready for merging far as I can see! 🚀
